### ISSUE TITLE
Add support for Red Dead Redemption 2

### ISSUE THREAD

**Issue by OsoaGH** [ModOrganizer2/modorganizer#1702]
**This template is useful to add basic support using https://github.com/ModOrganizer2/modorganizer-basic_games.** 

**If you are vaguely familiar with programming you can try following the instructions on that link to get something working yourself. If you have trouble with that please fill in this template.**

**Please provide the following information for the game (remove the optional lines you do not use):**

- **Name:** Red Dead Redemption 2
- **Short name:** reddeadredemption2
- **Nexus ID [optional]:** reddeadredemption2 (I think)
- **Executable:** RDR2.exe
- **Launcher [optional]:** Rockstar Launcher
- **Data path:** /
- **Documents path:** Documents\Rockstar Games\Red Dead Redemption 2
- **Saves directory [optional]:** Documents\Rockstar Games\Red Dead Redemption 2\Profiles\xxxxxxxx
- **Save extension [optional]:** .
- **Steam ID [optional]:** 1174180

**Provide additional details regarding the game that could be required for the plugin.**
Most script mods are installed in the root of the install directory. Lenny's Mod Loader mods are stored in the LML folder.

**Comment by rafaaelalves** [comment]
please, we need this!

**Comment by dirtydanisreal** [comment]
Yes! And some dlls go in scripts and some have their own folder. 

I'd love to be able to get the LML Mod Manager to load as an executable so we can control load order
